Add setting BMC roles to upgrade instructions (#1429)
Cray-HPE/docs-csm: upgrade/1.2/Stage_0_Prerequisites.md

To prevent any possibility of losing Workload Manager configuration data or files, a back-up is required. Please execute all Backup procedures (for the Workload Manager in use) located in the `Troubleshooting and Administrative Tasks` sub-section of the `Install a Workload Manager` section of the `HPE Cray Programming Environment Installation Guide: CSM on HPE Cray EX`. The resulting backup data should be stored in a safe location off of the system.

<a name="lock-ncn-bmcs"></a>

## Stage 0.6 - Verify NCN BMC Roles and Locks

Both NCNs and their BMCs require their roles to be set to 'Management' in HSM for admin locking purposes. This is done automatically for NCNs but manually for their BMCs. For BMCs, this role may not have been set during the previous installation.

> 1. Run the following command to ensure that the roles are properly set.

Reviewer: ```suggestion
1. Ensure that the roles in HSM are properly set.
```

    ```
    ncn-m001# cray hsm state components bulkRole update --role Management --component-ids $(cray hsm state components list --role management --type node --format json | jq -r .Components[].ID | sed 's/n[0-9]*//' | tr '\n' ',' | sed 's/.$//')
    ncn-m001# cray hsm state components query create --format json --component-ids $(cray hsm state components list --role management --type node --format json | jq -r .Components[].ID | sed 's/n[0-9]*//' | tr '\n' ',' | sed 's/.$//') | jq '.Components[] | "\(.ID) Role=\(.Role)"'
    "x3000c0s8b0 Role=Management"
    "x3000c0s9b0 Role=Management"
    "x3000c0s2b0 Role=Management"
    "x3000c0s6b0 Role=Management"
    "x3000c0s3b0 Role=Management"
    "x3000c0s7b0 Role=Management"
    "x3000c0s4b0 Role=Management"
    "x3000c0s5b0 Role=Management"
    ```

> 1. Run the following command to ensure they're all locked (NCNs and their BMCs).

Reviewer: ```suggestion
1. Ensure the NCNs and their BMCs are all locked.
```

    ```
    ncn-m001# /opt/cray/csm/scripts/admin_access/lock_management_nodes.py
    ```

> For more information about setting NCN BMC roles see [Set BMC Management Role](../../operations/hardware_state_manager/Set_BMC_Management_Role.md)

Reviewer: ```suggestion
For more information about setting NCN BMC roles, see [Set BMC Management Role](../../operations/hardware_state_manager/Set_BMC_Management_Role.md).
```

> For more information about locking NCNs and their BMCs see [Lock and Unlock Nodes](../../operations/hardware_state_manager/Lock_and_Unlock_Management_Nodes.md)

Reviewer: ```suggestion
For more information about locking NCNs and their BMCs, see [Lock and Unlock Nodes](../../operations/hardware_state_manager/Lock_and_Unlock_Management_Nodes.md).
```
